src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/BaseIntegrationTest.java

```diff
@@ -38,8 +38,7 @@
     protected static final String USER_ID = "1";
     protected static final String JURISDICTION_ID = "CMC";
     protected static final String CASE_TYPE_ID = "MoneyClaimCase";
-    protected static final String SUBMIT_CLAIM_EVENT = "submitClaimEvent";
-    protected static final String SUBMIT_POST_PAYMENT = "SubmitPostPayment";
+    protected static final String CREATE_NEW_CASE = "IssueClaim";
```

Comment:
may be we can remove SUBMIT_PRE_PAYMENT if it's not needed

Reply:
Done